src/memote_webservice/resources/submit.py

```diff
@@ -118,3 +119,10 @@ def _decompress(filename, content):
         else:
             content = BytesIO(content.read())
         return filename, content
+
+    @staticmethod
+    def _dump_model(filename, content):
+        uuid = str(uuid4())
+        LOGGER.warning(f"Dumping uploaded model '{filename}' to file '{uuid}'")
+        with open(uuid, 'wb') as f:
```

Comment:
For simplicity I'd like to keep the format of the uploaded file. So how about:
```python
with open(f"{uuid}_{filename}", 'wb') as f:
```
Also, slight :frowning_man: for single letter variable :stuck_out_tongue_winking_eye:

Reply:
I would advise against this. Allowing users to construct the filename may pose vulnerability issues for maliciously constructed filenames.

Reply:
https://tedboy.github.io/flask/generated/werkzeug.secure_filename.html